prow/config.yaml

```diff
@@ -26,6 +26,22 @@ sinker:
   max_pod_age: 6h
 
 deck:
+  spyglass:
```

Comment:
Will address the plank todo above (which makes the github status link go to spyglass) once we validate this is working